src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -80,6 +80,7 @@ class CrudField {
             if(this.isSubfield) {
                 window.crud.subfieldsCallbacks[this.parent.name] ??= [];
                 window.crud.subfieldsCallbacks[this.parent.name].push({ closure, field: this });
+                this.wrapper.trigger('CrudField:callbacksUpdated');
```

Comment:
Nitpicking, but wouldn't this be a bit clearer?

```suggestion
                this.wrapper.trigger('CrudField:subfieldCallbacksUpdated');
```

Reply:
I agree 👌